### PR TITLE
DAOS-3673 test: Verify container delete returns all space to be used again

### PR DESCRIPTION
**Pull request by Puspalogan**
- Created container/multiple_container_delete test to cover
    DAOS-3673 story
  - Modified util/ior_utils.py to handle ior 'dfs' api
  - Fixed a bug in util/ior_utils.py to return the environment
    without populating additional keys that are not necessary
    for POSIX api.

Signed-off-by: Logan Sundaram <logan.sundaram@intel.com>

### CODE REVIEW

File: src/tests/ftest/container/multiple_container_delete.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        self.assertTrue(final_ssd_fs == initial_ssd_fs)
+
+        # Uncomment the below verification once DAOS-3849 is fixed
+        #self.log.info("Verifying SCM space is recovered")
+        #self.log.info("{} == {}".format(final_scm_fs, initial_scm_fs)))
+        #self.assertTrue(final_scm_fs == initial_scm_fs)
+
+    def get_pool_space(self):
+        """Get pool free space for a given storage index
+        Args:
```
Comment (shimizukko) [inline src/tests/ftest/container/multiple_container_delete.py:95]: There's no argument, so we don't need this description?

Comment (Puspalogan) [reply src/tests/ftest/container/multiple_container_delete.py:95]: Agreed. Will fix it.

File: src/tests/ftest/container/multiple_container_delete.py [hunk 2]
```diff
@@ -91,12 +91,10 @@ def test_multiple_container_delete(self):
         #self.assertTrue(final_scm_fs == initial_scm_fs)
 
     def get_pool_space(self):
-        """Get pool free space for a given storage index
-        Args:
-            storage_index (int): pool property name. Defaults to
-                None.
+        """Get sscm and ssd pool free space
```
Comment (shimizukko) [inline src/tests/ftest/container/multiple_container_delete.py:94]:
```suggestion
        """Get scm and ssd pool free space
```
Typo?

Comment (Puspalogan) [reply src/tests/ftest/container/multiple_container_delete.py:94]: Yes, will fix it in another ticket of mine. Don't want to run the whole build for a minor change.

**Pull request merged**